incubator/percona-xtradb-cluster/values.yaml

```diff
@@ -0,0 +1,77 @@
+# Default values for Percona XtraDB Cluster
+
+## percona image and version
+## ref: https://hub.docker.com/r/percona/percona-xtradb-cluster/tags/
+imageRepo: "percona/percona-xtradb-cluster"
+imageTag: "5.7.19"
+replicas: 3
+
+## Specify password for root user
+##
+mysqlRootPassword: not-a-secure-password
```

Comment:
Passwords are auto-generated if not specified. However, this will never be the case, since you add a default here. Default should be auto-generated.